blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -6,6 +6,8 @@
 import hudson.model.User;
 import hudson.tasks.Mailer;
 import hudson.tasks.UserAvatarResolver;
+import hudson.util.HttpResponses;
```

Comment:
Right, I'm a little skeptical about that; the BlueUser seems like a data holder, I'm not a fan of mixing service methods with data classes. But this is what we have today, so; I'll defer this until it's needed for some reason, if that's ok.